x-pack/plugins/canvas/public/components/asset_manager/asset_manager.js

```diff
@@ -201,9 +196,8 @@ export class AssetManager extends React.PureComponent {
           <EuiModalBody>
             <EuiText size="s" color="subdued">
               <p>
-                Below are the image assets that you added to this workpad. To reclaim space, delete
-                assets that you no longer need. Unfortunately, any assets that are actually in use
-                cannot be determined at this time.
+                Below are image assets that have been added to this workpad. Any assets that are
```

Comment:
I prefer the wording "Below are image assets that you added to this workpad."

Reply:
I wonder about using "you" and "your" in Kibana since users essentially share everything on an instance. One person might have created the workpad, but I can log in, see it, and edit it.

Reply:
In this case, use either

Below are the image assets that were added to this workpad 

Or, shorter

Below are the image assets in this workpad.

Reply:
Thanks Gail!